class-00/lab-a/let-const/js/app.js

```diff
@@ -30,18 +30,18 @@ function makeRandom() {
 
 function displayPics(){
   while(viewed.length < 6){
-    var rando = makeRandom();
+    const rando = makeRandom();
     while(!viewed.includes(rando)){
       viewed.push(rando);
     }
   }
-  console.log(rando);
+  //console.log(rando);
   // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the variable declaration from `var to `let`.
-  // PUT YOUR RESPONSE IN THIS COMMENT
+  /* Var is a global variable and let is not a global variable so when we changed from var to let, the let in the rando function could not handle the let declaration */
```

Comment:
This isn't quite right! Using var doesn't automatically make something a global variable. In this case, when var was used on line 33, it did not create a global variable.